### ISSUE TITLE
l10n: relying on pasted strings (`add.loadingText`) is hazardous

### ISSUE THREAD

**Issue by jsoref** [rancher-sandbox/rancher-desktop#1481]
https://github.com/rancher-sandbox/rancher-desktop/blob/49d735b57ce64709fc5dba5dca3af1bf86eb74e3/src/assets/translations/en-us.yaml#L204

> if action is `stop`, then that'd be `stoping` instead of `stopping`.
> if it's `delete`, that'd be `deleteing` instead of `deleting`.

(Current values are `build` and `pull`, so `en-US` isn't harmed by this.)

Anyway, general localization rules is not to rely on pasting strings together as it can result in bad outcomes.

And different languages may need to put things in different places / etc., or conjugate differently, etc.

**Comment by ericpromislow** [comment]
Yes, best way to do `i18n` is to store full templatized phrases, and not assemble phrases or sentences as above. Marking as an enhancement (because nothing is actually broken right now).

**Comment by jandubois** [comment]
Fixed by bbdaeb90f97f722c56ef350d48844f6f8484f0ca (in #10637)
